app/uk/gov/hmrc/thirdpartydeveloperfrontend/service/ApplicationActionService.scala

```diff
@@ -57,48 +57,46 @@ class ApplicationActionService @Inject() (
   def toApiSubscriptionStatusList(
       application: ApplicationWithSubscriptionData,
       subscriptionFieldDefinitions: Map[ApiContext, Map[ApiVersionNbr, Map[FieldName, SubscriptionFieldDefinition]]],
-      summaryApiDefinitions: Map[ApiContext, ApiData]
+      summaryApiDefinitions: List[ApiDefinition]
     ): List[APISubscriptionStatus] = {
 
-    def handleContext(context: ApiContext, cdata: ApiData): List[APISubscriptionStatus] = {
-      def handleVersion(version: ApiVersionNbr, vdata: ApiVersion): APISubscriptionStatus = {
+    def handleContext(apiDefinition: ApiDefinition): List[APISubscriptionStatus] = {
+      def handleVersion(apiVersionNbr: ApiVersionNbr, apiVersion: ApiVersion): APISubscriptionStatus = {
```

Comment:
apiVersionNbr is available in ApiVersion so param is redundant - probably easiest to add a line like val apiVersionNbr = apiVersion.versionNbr rather than changing all the subsequent code.